### ISSUE TITLE
[enhancement]: Add border to the control image preview.

### ISSUE THREAD

**Issue by StellarBeing25** [invoke-ai/InvokeAI#6699]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

Currently there is no border to the control image preview, due to which the UI looks a little messed up depending upon the ratio of the uploaded image. Also, sometimes the buttons overlayed over bright images are not easily visible.

There should be a clear border present around the uploaded control or upscale image preview, and the overlayed buttons should be moved outside of the border to make the UI look more polished. The buttons should always be present and just greyed out when no image is uploaded. 

![Concept](https://github.com/user-attachments/assets/7ef09c10-8fcd-4afc-9cae-6eacc8f58922)

**Comment by psychedelicious** [comment]
Agreed it would look better w/ a border. I like how the action icons are not overlaid but instead in their on spot off to the side - easier to see them.
